### ISSUE TITLE
Ruby error messages

### ISSUE THREAD

**Issue by DanL12186** [Codewars/codewars-runner-cli#631]
Ever since (I believe) the new Coderunner was introduced, Most of the time Ruby error messages have become close to meaningless. I have to resort to running code through IRB in my terminal to get a meaningful error.

This can be pretty frustrating, especially for more difficult kata.

**Comment by kazk** [comment]
Can you elaborate? I don’t think anything was changed with new runner for Ruby.

**Comment by kazk** [comment]
It'll be helpful if you can show me some examples of meaningless errors and what you expect (or what you think it showed in the old runner).

**Comment by DanL12186** [comment]
Yeah, sorry. For some reason I had the idea that this was already well-known and I was just posting this as a 'reminder'.

I should have been clearer; it's not always bad, but a lot of the time when something goes wrong (I think maybe with method errors?) you get useless error messages.

I'll get some useful examples together before too long. Sorry about that :)

**Comment by kazk** [comment]
> For some reason I had the idea that this was already well-known

Is it this one? https://github.com/Codewars/codewars-runner-cli/issues/484
That one is caused by the message misinterpreted as HTML and wasn't introduced by the new runner.

**Comment by kazk** [comment]
No problem. We need to fix that. Thanks for reminding.
